src/server/services/persist-upload.js

```diff
@@ -83,4 +68,17 @@ async function persistUpload ({ filename, user, buffer }) {
   return upload
 }
 
-module.exports = { persistUpload, ValidationError, uploadFSName }
+async function bufferForUpload (upload) {
+  const data = await readFile(uploadFSName(upload))
+  return Buffer.from(data, 'binary')
+}
+
+async function documentsForUpload (upload) {
```

Comment:
I think we should try to rename any remaining references from "documents" to "records".

I guess it probably makes more sense to make that a follow-up task though.

Reply:
well, it now actually is returning documents, though maybe it could be `filesForUpload`